pkg/dyninst/symdb/func_name_test.go

```diff
@@ -0,0 +1,52 @@
+package symdb_test
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/dyninst/symdb"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
```

Comment:
linter will yell about your import ordering. Here's the vscode setting to have gopls fix 'em up automatically:
```
"gopls": {
        "formatting.local": "github.com/DataDog/datadog-agent",
},
```